subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.h

```diff
@@ -14,7 +14,10 @@ void radio_setup(void);
 void radio_reset(void);
 void radio_phy_set(u8_t phy, u8_t flags);
 void radio_tx_power_set(u32_t power);
+u32_t radio_tx_power_get(void);
 void radio_tx_power_max_set(void);
+u32_t radio_tx_power_min_get(void);
```

Comment:
Accept the lll->radio->hal_radio call path for radio_tx_power_min_get and radio_tx_power_max_get.